packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/calibrate-operation.ts

```diff
@@ -7,6 +7,7 @@ export interface CalibrationOperationStateCiemss extends BaseState {
 	chartConfigs: string[][];
 	mapping: CalibrateMap[];
 	simulationsInProgress: string[];
+	lossValues?: { [key: string]: number }[];
```

Comment:
Let's not store lossValues into the state, it makes the state and workflow object too large and unwieldy for other things.

Reply:
will do